### PR TITLE
Refactor cli input

### PR DESCRIPTION
**Pull request by Bandov**
Added the edit,addAttribute and deleteAttribute multiple attributes functionality

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditAttributeCommand.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                    }
+                } catch (NumberFormatException e) {
+                    throw new CommandException("Phone number for " + attributeName + " must be a number.");
+                }
+                attribute = new PhoneNumberAttribute("Phone", phoneNumber);
+                break;
+            default:
+                attribute = Attribute.fromString(attributeName, attributeValue);
+            }
+
```
Comment (ReflectiveObsidian) [inline src/main/java/seedu/address/logic/commands/EditAttributeCommand.java]: Add and Edit Attribute commands share this same logic, and eventually add person command will need to be updated to do this too. So could this part be extracted to some kind of Util class? Maybe something like "AttributeUtil".

File: src/main/java/seedu/address/logic/parser/DeleteAttributeCommandParser.java [hunk 2]
```diff
@@ -12,30 +14,44 @@ public class DeleteAttributeCommandParser implements Parser<DeleteAttributeComma
...
+
+        // Split the arguments string on spaces after the first "/"
+        String[] parts = args.split(" ", 2);
+
+        // Validate command structure
         if (parts.length < 2) {
-            throw new ParseException(Messages.MESSAGE_MISSING_ATTRIBUTES
-                    + "\n" + DeleteAttributeCommand.MESSAGE_USAGE);
+            throw new ParseException("Invalid command format. Expected format: "
+                    + "deleteAttribute /UUID /attributeName1 /attributeName2 ...");
```
Comment (ReflectiveObsidian) [inline src/main/java/seedu/address/logic/parser/DeleteAttributeCommandParser.java]: Could this use a constant variable , like was done previously?

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+     */
+    public EditAttributeCommand parse(String args) throws ParseException {
+        args = args.trim();
+
+        // Split the arguments string on spaces after the first "/"
+        String[] parts = args.split(" ", 3);
+
+        // Validate command structure
+        if (parts.length < 3 || !"editAttribute".equalsIgnoreCase(parts[0])) {
+            throw new ParseException("Invalid command format. Expected format: editAttribute /uuid [attributes]");
```
Comment (ReflectiveObsidian) [inline src/main/java/seedu/address/logic/parser/EditCommandParser.java]: Could this use MESSAGE_USAGE, or some other constant?

File: src/main/java/seedu/address/logic/parser/PersonAttributeCommandParser.java [hunk 4]
```diff
@@ -1,13 +1,17 @@
 package seedu.address.logic.parser;
```
Comment (ReflectiveObsidian) [inline src/main/java/seedu/address/logic/parser/PersonAttributeCommandParser.java]: Could this be renamed to AddAttributeParser, and the common logic extracted to ParserUtil?

File: src/main/java/seedu/address/logic/commands/EditAttributeCommand.java (hunk 1 above)
Comment (Bandov) [reply src/main/java/seedu/address/logic/commands/EditAttributeCommand.java]: sure

File: src/main/java/seedu/address/logic/parser/DeleteAttributeCommandParser.java (hunk 2 above)
Comment (Bandov) [reply src/main/java/seedu/address/logic/parser/DeleteAttributeCommandParser.java]: done

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java (hunk 3 above)
Comment (Bandov) [reply src/main/java/seedu/address/logic/parser/EditCommandParser.java]: yep

File: src/main/java/seedu/address/logic/parser/PersonAttributeCommandParser.java (hunk 4 above)
Comment (Bandov) [reply src/main/java/seedu/address/logic/parser/PersonAttributeCommandParser.java]: yep

**Review by ReflectiveObsidian — APPROVED** [review]
LGTM

**Pull request merged**